### PR TITLE
added icons to the login component

### PR DESCRIPTION
**Pull request by Oyinbrakemigrace**
added icons to the login component

### CODE REVIEW

**Comment by Oyinbrakemigrace** [comment]
loginCompScreenshot

File: src/components/userLogin_comp.js [hunk 1]
```diff
@@ -12,9 +16,19 @@ function UserLoginComp() {
 
         <div className="flex justify-between items-center">
           <button className="bg-gray-200 flex items-center py-2 px-10 mx-3 rounded-md">
```
Comment (annonymousauthority) [inline src/components/userLogin_comp.js:18]: Give this a `w-1/3` and also the button at line 26

File: src/components/userLogin_comp.js [hunk 2]
```diff
@@ -12,9 +16,19 @@ function UserLoginComp() {
 
         <div className="flex justify-between items-center">
```
Comment (annonymousauthority) [inline src/components/userLogin_comp.js:17]: Give this a `justify-center` and `space-x-12`

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Some changes to your design.

**Comment by Oyinbrakemigrace** [comment]
loginCompScreenshot
New screenshot for the changes added to the style

**Comment by Oyinbrakemigrace** [comment]
loginCompScreenshot
Aligned loginComponent

**Review by annonymousauthority — APPROVED** [review]
LG

**Pull request merged**